### ISSUE TITLE
Implement a way to identify fixtures outside of `tests/prague/eip7692_eof_v1` that test EOF

### ISSUE THREAD

**Issue by marioevz** [ethereum/execution-spec-tests#767]
Raised by @pdobacz in https://github.com/ethereum/execution-spec-tests/pull/754#issuecomment-2305048537, we need a way to efficiently select tests that verify EOF and are outside of the main tests folder (`tests/prague/eip7692_eof_v1`) but still produce EOF V1 code.

E.g. tests that use the `@pytest.mark.with_all_evm_code_types` decorator to automatically fill in both legacy and EOF modes, because these tests remain in the respective EIP folder but also exercise EOF code paths.

**Comment by pdobacz** [comment]
Hey, do I understand correctly, that before we solve this there is no way to include such tests (EOF-wrapped MCOPY tests for example, as the PR linked) in the [`eip7692` pre-releases](https://github.com/ethereum/execution-spec-tests/releases/tag/eip7692%40v2.0.0)?

Could we just put them under the `state_tests/cancun/eip5656_mcopy`? as long as they're in the eip7692 pre-release and consumers just run everything in state_tests, and the actual target fork (`Osaka`) is in the JSON anyway, that could work maybe?

**Comment by marioevz** [comment]
> Could we just put them under the `state_tests/cancun/eip5656_mcopy`? as long as they're in the eip7692 pre-release and consumers just run everything in state_tests, and the actual target fork (`Osaka`) is in the JSON anyway, that could work maybe?

Yes I think this could be the quick solution to this, simply fill everything using the `Osaka` fork.

**Comment by felix314159** [comment]
Re-open when EOF becomes relevant again
